docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -16,71 +16,95 @@ MATCH <pattern> [<WHERE clause>] RETURN <output>
 
 ## The workflow of MATCH
 
-1. The `MATCH` statement uses a **native index** to locate a source vertex. The vertex can be in any position in a pattern. In other words, in a valid `MATCH` statement, **there must be an indexed property or tag, or a specific VID.** For how to index a property, see [Create native index](../14.native-index-statements/1.create-native-index.md).
+1. The `MATCH` statement uses a native index to locate a source vertex or an edge. The source vertex or the edge can be in any position in the pattern. In other words, in a valid `MATCH` statement, **there must be an indexed property, a tag, or an edge type. Or the VID of a specific vertex must be specified with the id() function in the `WHERE` clause.** For how to create an index, see [create native index](../14.native-index-statements/1.create-native-index.md).
 
-  !!! note
+2. The `MATCH` statement searches through the pattern to match edges or vertices.
 
-        The native index for VID is created by default, so you don't need to create an extra index if you want to match on VID.
+  !!! note
 
-2. The `MATCH` statement searches through the pattern to match edges and other vertices.
+        The path type of the `MATCH` statement is `trail`. That is, only vertices can be repeatedly visited in the graph traversal. Edges cannot be repeatedly visited. For details, see [path](../../1.introduction/2.1.path.md).
 
 3. The `MATCH` statement retrieves data according to the `RETURN` clause.
 
-!!! note "OpenCypher compatibility"
+## OpenCypher compatibility
+
+- For now, nGQL does not support traversing all vertices and edges with `MATCH`, such as `MATCH (v) RETURN v`. However, after the index of a certain tag is created, all corresponding vertices can be traversed, such as `MATCH (v:T1) RETURN v`.
 
-    For now, nGQL DOES NOT support scanning all vertices and edges with `MATCH`. For example, `MATCH (v) RETURN v`.
+- Graph pattern is not supported in the `WHERE` clause.
 
-## Use patterns in MATCH statements
+## Patterns
```

Comment:
Consider restoring this line. Otherwise, it looks like this section introduces patterns.